### ISSUE TITLE
Arguments `target` and `output` must have the same shape. Received: target.shape=(32, 4), output.shape=(32, 1)

### ISSUE THREAD

**Issue by kequeen** [fchollet/deep-learning-with-python-notebooks#246]
`model.compile(optimizer="rmsprop",
              loss={"priority": "mean_squared_error", "department": "categorical_crossentropy"},
              metrics={"priority": ["mean_absolute_error"], "department": ["accuracy"]})
model.fit({"title": title_data, "text_body": text_body_data, "tags": tags_data},
          {"priority": priority_data, "department": department_data},
          epochs=1)
model.evaluate({"title": title_data, "text_body": text_body_data, "tags": tags_data},
               {"priority": priority_data, "department": department_data})
priority_preds, department_preds = model.predict(
    {"title": title_data, "text_body": text_body_data, "tags": tags_data})`

something is wrong in this code
![image](https://github.com/user-attachments/assets/84d56158-09b0-42c7-94b7-b0fc9f6c18c4)
